lib/ui/text.dart

```diff
@@ -56,7 +56,15 @@ class FontWeight {
 
   /// A list of all the font weights.
   static const List<FontWeight> values = <FontWeight>[
-    w100, w200, w300, w400, w500, w600, w700, w800, w900
+    w100,
```

Comment:
Can the reformatting of this file be undone?

I'd prefer to keep this PR limited to FFI-related changes